src/containers/JudgingPanel.js

```diff
@@ -46,16 +47,67 @@ class Project {
   }
 }
 
+const getGradedProjects = async () => {
+  const projectDocs = await projectsRef.get()
+  const projectData = projectDocs.docs.map(projectDoc => {
+    const project = projectDoc.data()
+    if (project.grades) {
+      project.countGraded = Object.values(project.grades).length
+      Object.values(project.grades).forEach(grade => {
+        Object.entries(grade).forEach(([key, value]) => {
+          if (key === 'notes') return
+          project[key] = project[key] ? project[key] + value : value
+          project.total = project.total ? project.total + value : value
+        })
+      })
+      project.grade = project.total / project.countGraded / 5
+      project.tech = project.tech / project.countGraded
+      project.design = project.design / project.countGraded
+      project.functionality = project.functionality / project.countGraded
+      project.creativity = project.creativity / project.countGraded
+      project.pitch = project.pitch / project.countGraded
+    } else {
+      project.countGraded = 0
+      project.grade = 0
+    }
+    return project
+  })
+  projectData.sort((a, b) => {
+    if (a.grade > b.grade) {
+      return -1
+    }
+    if (a.grade < b.grade) {
+      return 1
+    }
+    return 0
```

Comment:
pretty sure you can just return `a.grade - b.grade`

Reply:
@jackyzha0 i think u mean `b.grade - a.grade` according to the logic here 
but ya @ianmah `.sort()` only cares about positive vs negative, doesn't have to be -1/0/1

Reply:
this pr is giving me major 310 flashbacks

Reply:
lmao truth 
tbh these are minor comments, i think it's good to merge (up to u if u wanna make refactor issues so we dont' forget -- or so we can close the refactor issues later if we decide not to do it)